scripts/trias/run_trias.sh

```diff
@@ -0,0 +1,171 @@
+cd /opt/go/src/trias/
+git clone https://${USER_NAME}:${PASSWD}@github.com/trias-lab/trias-application.git --depth 1
+cd /opt/go/
+go build -o bin/triascode_app src/trias/trias-application/cmd/triascode_app.go
+
+# 7 install blackbox & blackbox_agent
+mkdir -p /opt/python
+cd /opt/python
+cp /opt/core . -r
+cd /opt/python/core/TCserver/worker
+cp -av Blackbox_global.py blackbox_global.py
+rm -rf build/
+apt-get install -y --force-yes libatlas-base-dev
+pyinstaller --clean -F blackbox_main.py
+mv dist/blackbox_main dist/blackbox
+
+cd /opt/python/core/TCserver/worker/blackbox_agent
+pyinstaller --clean -p /opt/python/core/TCserver/worker -F blackbox_agent.py
+
+# 8 download attestation
+cd /opt/python
+git clone https://${USER_NAME}:${PASSWD}@github.com/trias-lab/attestation
+
+# 9 modify grub to support IMA
+sed -i "/linux\t/s/$/& ima_tcb ima_template=\"ima\" ima_hash=\"sha1\"/g" /boot/grub/grub.cfg
+
+# 10 install tpm-emulator-master
+cp -R /opt/core/deploy/file/tpm/tpm-emulator-master /
+cd /tpm-emulator-master
+sh /tpm-emulator-master/install.sh
+
+# 11 PREPARE FOR TAKEOFF...
+# add user 'ubuntu', why???
+useradd -m -s /bin/bash ubuntu
+
+## copy 'tendermint' and 'triascode_app'
+cp /opt/go/bin/triascode_app /usr/local/bin/ && chown ubuntu:ubuntu /usr/local/bin/triascode_app && chmod 755 /usr/local/bin/triascode_app
+cp /opt/go/bin/tendermint /usr/local/bin/ && chown ubuntu:ubuntu /usr/local/bin/tendermint && chmod 755 /usr/local/bin/tendermint
+
+mkdir -p /trias/log /trias/.ethermint/keystore /trias/.ethermint/tendermint
+chown -R ubuntu:ubuntu /trias
+
+## copy 'blackbox' and 'blackbox_agent'
+cp -R /opt/core/deploy/file/8lab /
+mkdir /8lab/log /8lab/conf /8lab/config
+
+cp -av /opt/python/core/TCserver/worker/dist/blackbox /8lab/
+cp -av /opt/python/core/TCserver/worker/blackbox_agent/dist/blackbox_agent /8lab/
+cp -R /opt/core/deploy/file/configure.json /8lab/conf/
+## change ip address
+IP=$(ip addr | grep 'state UP' -A2 | tail -n1 | awk '{print $2}' | cut -f1  -d'/')
+sed -i -r "s/(\"SeverIP\": \")[^\"]*/\1$IP/" /8lab/conf/configure.json
+sed -i -r "s/(\"RestIP\": \")[^\"]*/\1$IP/" /8lab/conf/configure.json
+sed -i -r "s/(\"TrueClientIP\": \")[^\"]*/\1$IP/" /8lab/conf/configure.json
+
+## copy attestation
+cp -R /opt/python/attestation /
+
+## copy the startup scripts:
+##  'Trias' starts up 'tendermint' and 'triascode_app',
+##  'start-tpmd' starts up tpmd and tcsd,
+##  'BlackBoxClient' starts up 'blackbox', 'blackbox_agent' and 'attestation'.
+cp /opt/core/deploy/file/Trias /etc/init.d/Trias && chmod 775 /etc/init.d/Trias
+cp /opt/core/deploy/file/start-tpmd /etc/init.d/start-tpmd && chmod 775 /etc/init.d/start-tpmd
+cp /opt/core/deploy/file/BlackBoxClient /etc/init.d/BlackBoxClient && chmod 775 /etc/init.d/BlackBoxClient
+
+# 12 Startup services
+update-rc.d BlackBoxClient defaults
+update-rc.d start-tpmd defaults
+update-rc.d Trias defaults
+# reboot
+
+# 13 Flying...
+/etc/init.d/start-tpmd start
+/etc/init.d/Trias start
```

Comment:
I was try to start 'trias_accs' in Trias and it seems well.
But I got this in log file.
`root@ubuntu:/trias/log# cat trias_accs.out
nohup: failed to run command 'trias_accs': No such file or directory`

I think there may be some steps missing.

Reply:
Ohh, they have just updated these scripts yesterday....
Try to get the earlier version of 'core' , my version is ee3a157be0c59c1a7edbf5b4cb0c5e105b79688a